examples/freertos/explorer_board/src/uart/uart_demo.h

```diff
@@ -0,0 +1,12 @@
+// Copyright 2022 XMOS LIMITED.
+// This Software is subject to the terms of the XMOS Public Licence: Version 1.
+
+#ifndef UART_DEMO_H_
+#define UART_DEMO_H_
+
+#include "FreeRTOS.h"
+
+void uart_demo_create(UBaseType_t priority);
```

Comment:
From what I can tell, this demo runs forever, even if it encounters errors.  That design is fine with me, but I think it does deserve some documentation somewhere that says so.